### PR TITLE
Updating jquery in Dev Portal

### PR DESCRIPTION
**Pull request by damianpm**
**What this PR does / why we need it**:

Updates jquery from 1.7 to 3.x in the developer portal, for security reasons.
It affects only new instances of the developer portal

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-5381

**Verification steps** 

To create a new instance of the developer portal, you can reset the database
`bundle exec rake db:reset MASTER_PASSWORD=p USER_PASSWORD=p`

**Special notes for your reviewer**:

### CODE REVIEW

File: lib/developer_portal/app/views/developer_portal/javascripts/3scale.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ var hljs=new function(){function e(e){return e.replace(/&/gm,"&amp;").replace(/<
  *
  * Requires jQuery 1.6.0 or later.
  * https://github.com/rails/jquery-ujs
-
```
Comment (didierofrivia) [inline lib/developer_portal/app/views/developer_portal/javascripts/3scale.js:28]: Avoid these kind of changes

File: app/assets/javascripts/provider/cms_toolbar_v2.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  })
+  $('#hide-side-bar').on('click', event => {
+    event.preventDefault()
+    toggleValues()
+    if (storedToolbarState() === 'hidden') {
+      saveToolbarState('visible')
+    } else {
+      saveToolbarState('hidden')
+    }
```
Comment (josemigallas) [inline app/assets/javascripts/provider/cms_toolbar_v2.js:49]: Should we modernize the code as well?
```suggestion
    saveToolbarState(storedToolbarState() === 'hidden' ? 'visible' : 'hidden')
```

Comment (damianpm) [reply app/assets/javascripts/provider/cms_toolbar_v2.js:49]: But then why "modernizing" this line and not the others? The whole file can be "modernized", but I'd do it as a separate task.

Comment (josemigallas) [reply app/assets/javascripts/provider/cms_toolbar_v2.js:49]: This was only an example, but yes let's leave it for later!

**Pull request merged**